Calibration/HcalCalibAlgos/plugins/HcalHBHEMuonAnalyzer.cc

```diff
@@ -1188,12 +1221,33 @@ double HcalHBHEMuonAnalyzer::activeLength(const DetId& id_) {
     dpths.emplace_back(id.depth());
   }
   double lx(0);
+  if (debug) {
+#ifdef EDM_ML_DEBUG
```

Comment:
Thanks @bsunanda for addressing my previous comment!
Looking again, I'm now a bit confused by this:
```
if(debug) {
#ifdef EDM_ML_DEBUG
...
}
```
What is the purpose of having two "debug safeguards" here?
Sorry for noticing this only now (and if the answer is obvious)....